compliance_suite/cli.py

```diff
@@ -51,8 +50,7 @@ def report(server: str,
         raise Exception("No server provided. Please provide a server URL.")
 
     if version is None:
-        logger.info("No version provided. Latest version is used as default value.")
-        version = "v1.0"                    # Hardcode the latest version here. TODO - Future versions
+        raise Exception("No version provided. Please provide a version.")
```

Comment:
I actually think defaulting to the latest version is a reasonable behavior. Or was there anything that speaks against it?

Reply:
Adding a version will make it TES-specific. As an OpenAPI runner, multiple services will be run and we cant track their latest version.